README.md

```diff
@@ -101,7 +101,7 @@ and use it on your hosts by running:
 
 ```sh
 curl -sSL https://dl.signalfx.com/signalfx-agent.sh > /tmp/signalfx-agent.sh
-sudo sh /tmp/signalfx-agent.sh YOUR_SIGNALFX_API_TOKEN --realm YOUR_SIGNALFX_REALM
+sudo sh /tmp/signalfx-agent.sh -- YOUR_SIGNALFX_API_TOKEN --realm YOUR_SIGNALFX_REALM
```

Comment:
Make this like the quick-install.md one.  Normally the `--` comes _after_ all other arguments, not in the middle.

Reply:
done, PR is updated